vendor/modules.txt

```diff
@@ -919,10 +926,15 @@ github.com/keybase/go-crypto/rsa
 github.com/konsorten/go-windows-terminal-sequences
 # github.com/kr/fs v0.1.0
 github.com/kr/fs
+<<<<<<< HEAD
```

Comment:
Looks like there's still an unresolved conflict

Reply:
just run go mod tidy && go mod vendor instead of trying to solve the merge conflict.. maybe?

Reply:
Sorry my fault I'll do it asap